package corejava;

public class ExceptionHandle {
	public static void main(String args[]) {
	try {
		
	 int i = 0;
	 int j = 10/i;
	 System.out.println(j);
		}
	catch(ArithmeticException e)	{
		
		System.out.println("can not be divided by 0");
	}
	
	System.out.println("After Exception");
	
	}
    }

